README.md

```diff
@@ -591,13 +591,7 @@ Parameters:
 
 ### VimScript
 
-> **Note:** Since 1.0.0, vimscript functions are deprecated. Use the [Lua](#%EF%B8%8F-api) api instead.
-
--   `VSCodeNotify()`/`VSCodeCall()`: deprecated, use [Lua](#%EF%B8%8F-api) `require('vscode').call()` instead.
--   `VSCodeNotifyRange()`/`VSCodeCallRange()`: deprecated, use [Lua](#%EF%B8%8F-api)
-    `require('vscode').call(…, {range:…})` instead.
--   `VSCodeNotifyRangePos()`/`VSCodeCallRangePos()`: deprecated, use [Lua](#%EF%B8%8F-api)
-    `require('vscode').call(…, {range:…})` instead.
+You can use `v:lua.require("vscode")` to access the API from VimScript.
```

Comment:
The new text removed the link to the Lua text. And now ctrl-f on the readme won't find "VSCodeNotify".

I'm always in favor of removing tech debt, but this section is very small and minimal, thus low cost. It takes a long time for old blog posts, etc, to fade away. And we haven't removed the actual code. So this readme change is a regression.

Reply:
makes sense